.travis.yml

```diff
@@ -1,5 +1,3 @@
-sudo: required # needed for docker
```

Comment:
It turns out that Travis doesn't need this to run a docker container.

As a result, the build process is much faster, from > 3 minutes to slightly over one minute.